### PR TITLE
Customers v2

### PR DESCRIPTION
**Pull request by claudiosanches**

### CODE REVIEW

File: source/includes/wp-api-v2/_customers.md [hunk 1]
```diff
@@ -4,29 +4,24 @@ The customer API allows you to create, view, update, and delete individual, or a
...
+|----------------------|-----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `id`                 | integer   | Unique identifier for the resource. <i class="label label-info">read-only</i>                                                                                                                                                                                                                                                                                      |
+| `date_created`       | date-time | The date the customer was created, in the site's timezone. <i class="label label-info">read-only</i>                                                                                                                                                                                                                                                               |
```
Comment (mikejolley) [inline source/includes/wp-api-v2/_customers.md]: Confirm not UTC.

File: source/includes/wp-api-v2/_customers.md [hunk 2]
```diff
@@ -4,29 +4,24 @@ The customer API allows you to create, view, update, and delete individual, or a
...
+| `last_name`          | string    | Customer last name.                                                                                                                                                                                                                                                                                                                                                |
+| `role`               | string    | Customer role. <i class="label label-info">read-only</i>                                                                                                                                                                                                                                                                                                           |
+| `username`           | string    | Customer login name. Can be generated automatically from the customer's email address if the option `woocommerce_registration_generate_username` is equal to `yes` <i class="label label-info">cannot be changed</i> <i class="label label-info">maybe mandatory</i>                                                                                               |
```
Comment (justinshreve) [inline source/includes/wp-api-v2/_customers.md]: maybe mandatory is a little confusing - can we word these a different way?

Comment (claudiosanches) [inline source/includes/wp-api-v2/_customers.md]: It's maybe, but we say "Can be generated automatically from the customer's email address if the option `woocommerce_registration_generate_username` is equal to `yes`". At the same time, if you check the `OPTIONS` for it, will return if is mandatory or not on your site.

**Comment by mikejolley** [comment]
Closing this after our discussion this morning on generating docs.

**Pull request closed without merging**